cmd/manager/aggregator.go

```diff
@@ -301,13 +301,13 @@ type compResultIface interface {
 
 func createOrUpdateOneResult(crClient aggregatorCrClient, owner metav1.Object, labels map[string]string, annotations map[string]string, exists bool, res compResultIface) error {
 	kind := res.GetObjectKind()
-
 	if err := controllerutil.SetControllerReference(owner, res, crClient.getScheme()); err != nil {
 		cmdLog.Error(err, "Failed to set ownership", "kind", kind.GroupVersionKind().Kind)
 		return err
 	}
 
 	res.SetLabels(labels)
+	annotations = setScanTimestamp(crClient, owner, annotations)
```

Comment:
Seems like we have some inconsistency here in using receivers versus functions. What about:

```go
res.setTimestampAnnotations(...)
```

Or reusing the annotation function below?